lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -56,8 +56,11 @@ function BaseDialog({
         )
       }
     >
-      {children}
-      {error && <ErrorDisplay error={error} />}
+      {error ? (
+        <ErrorDisplay error={error}>{children}</ErrorDisplay>
+      ) : (
+        <>{children}</>
```

Comment:
```suggestion
        {children}
```

And possibly without curly braces:
```suggestion
        children
```